src/Extend/Middleware.php

```diff
@@ -97,5 +105,9 @@ public function extend(Container $container, Extension $extension = null)
 
             return $existingMiddleware;
         });
+
+        $container->extend("flarum.{$this->frontend}.csrfExemptRoutes", function ($existingExemptRoutes) {
+            return $existingExemptRoutes + $this->csrfExemptRoutes;
```

Comment:
The other additions to arrays happen via a `[]` method, I feel like we should keep that consistent across the extender/code unless there is a specific reason that you've chosen to use a `+` here.

```suggestion
            return $existingExemptRoutes[] = $this->csrfExemptRoutes;
```

Reply:
After further discussion on discord I realized that there was no foreach loop here, And based on that and some testing I believe that using `array_merge` would be the ideal way of doing this (best performance)

Reply:
Will do, thanks! Only thing is your change has ExemptRules, whereas should be exemptRoutes. I'll address this in a separate commit though

Reply:
Fixed in 15c08f6